index.html

```diff
@@ -47,7 +48,7 @@ <h1 class="u-section-title">La brasserie</h1>
       <h1 class="u-section-title">Nos bieres</h1>
       <div class="Beer Beer__GoldenAle">
         <div class="Beer-image">
-          <h2>Golden ale</h2>
+          <h2> <header> Blonde </header>Golden ale</h2>
```

Comment:
We usually don't use [header](https://developer.mozilla.org/fr/docs/Web/HTML/Element/header) like that